### ISSUE TITLE
ImportError: cannot import name 'decode_base64_to_file' from 'gradio.processing_utils'

### ISSUE THREAD

**Issue by BelieveDiffusion** [TheLastBen/fast-stable-diffusion#2083]
I opened TheLastBen colab today, and I'm getting the following error when I try to start Stable-Diffusion:

```
Error loading script: api.py
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/scripts.py", line 256, in load_scripts
    script_module = script_loading.load_module(scriptfile.path)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/script_loading.py", line 11, in load_module
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/api.py", line 15, in <module>
    from modules.api import api
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/api/api.py", line 9, in <module>
    from gradio.processing_utils import decode_base64_to_file
ImportError: cannot import name 'decode_base64_to_file' from 'gradio.processing_utils' (/usr/local/lib/python3.10/dist-packages/gradio/processing_utils.py)
```

The workbook was working great with the same config yesterday evening. I'm using https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb . (I don't have `Use_Latest_Working_Commit` checked.)

**Comment by sahildeshpande** [comment]
same

**Comment by BelieveDiffusion** [comment]
Yeah… not sure what's going on, but the notebook is unusable for me tonight. Fingers crossed for a quick fix!

I also tried running an earlier version of this notebook from before the changes made today (81d0bd31 with the `Use_Latest_Working_Commit` checkbox checked, but that didn't work either:

```
Error loading script: api.py
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/scripts.py", line 256, in load_scripts
    script_module = script_loading.load_module(scriptfile.path)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/script_loading.py", line 11, in load_module
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/api.py", line 15, in <module>
    from modules.api import api
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/api/api.py", line 25, in <module>
    from modules.sd_models import checkpoints_list, unload_model_weights, reload_model_weights
ImportError: cannot import name 'unload_model_weights' from 'modules.sd_models' (/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/sd_models.py)
```

…and then later…

```
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 340, in <module>
    webui()
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 243, in webui
    shared.demo = modules.ui.create_ui();shared.demo.queue(concurrency_count=999999,status_update_rate=0.1)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/ui.py", line 446, in create_ui
    modules.scripts.scripts_txt2img.initialize_scripts(is_img2img=False)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/scripts.py", line 298, in initialize_scripts
    script = script_class()
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet.py", line 229, in __init__
    self.txt2img_w_slider = gr.Slider()
  File "/usr/local/lib/python3.10/dist-packages/gradio/components.py", line 695, in __init__
    IOComponent.__init__(
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/scripts.py", line 544, in IOComponent_init
    add_classes_to_gradio_component(self)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/scripts.py", line 529, in add_classes_to_gradio_component
    comp.elem_classes = ["gradio-" + comp.get_block_name(), *(comp.elem_classes or [])]
AttributeError: 'Slider' object has no attribute 'elem_classes'
```

The tragedy is, I'm not even using Gradio… I have a free ngrok authtoken provided instead.

**Comment by sahildeshpande** [comment]
Got it to work with the latest commit check

**Comment by hansale** [comment]
Turn off `Use_Latest_Working_Commit` and it seems to be working.

**Comment by BelieveDiffusion** [comment]
Nope - I'm still getting `ImportError: cannot import name 'decode_base64_to_file' from 'gradio.processing_utils' (/usr/local/lib/python3.10/dist-packages/gradio/processing_utils.py)` :(

**Comment by BelieveDiffusion** [comment]
Tried it without the Ngrok token; still get the same error.

**Comment by sahildeshpande** [comment]
Worked for me after checking the box for Use_Latest_Working_Commit, and then running the install again.

**Comment by beanzie** [comment]
Everything was working fine for me yesterday without `Use_Latest_Working_Commit` checked. I noticed updates to the output resolution options for img2img.

Today it was still working, but SwinIR is mysteriously missing from extras/hiresfix. I attempted reinstalling with `Use_Latest_Working_Commit` checked, and got the errors mentioned above during launch.

Deleting the runtime and installing again with `Use_Latest_Working_Commit` unchecked, it works, still no SwinIR though
